packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -274,6 +280,27 @@ describe('reporter', () => {
         'network.client.ip': '8.8.8.8'
       })
     })
+
+    it('should call standalone sample if standalone ASM enabled', () => {
```

Comment:
if I understood it correctly, `standalone.sample()` should be sent always, not only if standalone ASM is enabled, and sample adds the tag or not depending if it is enabled or not